db/migrate/20250708065035_add_reminder_mails_active_to_companies.rb

```diff
@@ -0,0 +1,7 @@
+class AddReminderMailsActiveToCompanies < ActiveRecord::Migration[8.0]
+  def change
+    unless column_exists?(:companies, :reminder_mails_active)
```

Comment:
That should never be needed.
Either we are actually migrating and the column doesn't exist or something went horribly wrong :grimacing: